crates/core_simd/Cargo.toml

```diff
@@ -26,3 +26,6 @@ features = ["alloc"]
 
 [dev-dependencies.test_helpers]
 path = "../test_helpers"
+
+[dev-dependencies]
+std_float = { path = "../std_float/" }
```

Comment:
I think we want to reverse this dependency, since we might want to put more complex functions in std that use `core::simd`.  Those tests would need to be moved, of course.

Reply:
It already has a dependency flowing the other way, actually, which is why this is a dev dependency: Cargo would not accept it if it was a true circular dependency. So this is only linked for running tests andexamples currently. In `./std_float/Cargo.toml`:
```toml
[dependencies]
core_simd = { path = "../core_simd" }
```

Reply:
Got it--makes sense